src/Microsoft.DotNet.Wpf/src/PresentationBuildTasks/Microsoft/Build/Tasks/Windows/GenerateTemporaryTargetAssembly.cs

```diff
@@ -671,6 +674,110 @@ private void RemoveItemsByName(XmlDocument xmlProjectDoc, string sItemName)
 
         }
 
+        //
+        // Remove specific property from project file. Every property should be under an PropertyGroup.
+        //
+        private void RemovePropertiesByName(XmlDocument xmlProjectDoc, string sPropertyName)
```

Comment:
Since the only difference between `RemoveItemsByName` and `RemovePropertiesByName` is the rootnode name comparison, we can refactor it into a common method and pass it as a parameter.

Reply:
The reasoning behind making a separate function is to avoid modification of behavior and signature of existing function to avoid any unforeseen regressions.